addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -26,6 +26,9 @@ const GalleryWidget = publicWidget.Widget.extend({
      * @param {Event} ev
      */
     _onClickImg: function (ev) {
+        if (this.$modal && this.$modal.data('bs.modal') && this.$modal.data('bs.modal')._isShown) {
```

Comment:
It's not good relying on a detail implementation of BS4... the isShown variable starts with an underscore for a reason :grimacing: 

Just check `if (this.$modal) {` and unset the variable once the modal is closed.